source/solvers/navier_stokes_scratch_data.cc

```diff
@@ -364,5 +363,13 @@ NavierStokesScratchData<dim>::calculate_physical_properties()
     }
 }
 
+template <int dim>
+void
+NavierStokesScratchData<dim>::set_pressure_scaling_factor(
+  const double pressure_scaling_factor)
+{
+  this->pressure_scaling_factor = pressure_scaling_factor;
+}
```

Comment:
Why not pass the scaling factor to the constructor of the thing instead?

Reply:
I agree it would be cleaner this way. You mean the reinit of the constructor itself?